stages/org.osbuild.coreos.live-artifacts.mono.meta.json

```diff
@@ -0,0 +1,68 @@
+{
+  "summary": "Build CoreOS Live ISO and PXE kernel,initramfs,rootfs",
+  "description": [
+    "This stage builds the CoreOS Live ISO and PXE kernel,initramfs,rootfs",
+    "artifacts. Input to this stage are metal and metal4k raw disk images",
```

Comment:
```suggestion
    "artifacts. Inputs to this stage are metal and metal4k raw disk images",
```

Reply:
LGTM. I'll include it in the next upload.